src/clr-core/common/utils/global.ts

```diff
@@ -50,20 +54,28 @@ function getVersion() {
     userAgent: navigator.userAgent,
   };
 
-  console.log(JSON.stringify(log, null, 2));
+  if (console && console.log) {
+    console.log(JSON.stringify(log, null, 2));
+  }
   return log;
 }
 
 function getAngularVersion() {
-  const appRoot = document.querySelector('[ng-version]');
+  const appRoot = document && document.querySelector('[ng-version]');
   return appRoot ? appRoot.getAttribute('ng-version') : undefined;
 }
 
 function setRunningVersion() {
-  window.CDS._version = [...new Set([...window.CDS._version, '@VERSION'])];
+  const loadedVersion = '@VERSION';
+  if (!window) {
+    return;
+  }
+  if (window.CDS._version.indexOf(loadedVersion) < 0) {
+    window.CDS._version.push(loadedVersion);
+  }
   (document.querySelector('body') as HTMLElement).setAttribute('cds-version', window.CDS._version[0]);
 
-  if (window.CDS._version.length > 1) {
+  if (console && console.warn && window.CDS._version.length > 1) {
```

Comment:
I think console is available in standard node env so there is no need to check for it